apps/pollbook/backend/src/app_multi_node.test.ts

```diff
@@ -481,6 +482,49 @@ test('connection status is managed properly with many pollbooks', async () => {
         expect(context.peerWorkspace.store.getNewEvents).toHaveBeenCalled();
       }
     });
+
+    // If the pollbook package hash is different they will not connect
+    let i = 0;
+    for (const context of pollbookContexts) {
+      context.workspace.store.deleteElectionAndVoters();
+      context.workspace.store.setElectionAndVoters(
+        electionDefinition,
+        `fake-package-hash-${i}`,
+        testStreets,
+        testVoters
+      );
+      i += 1;
+    }
+    vitest.advanceTimersByTime(NETWORK_POLLING_INTERVAL);
+    await vi.waitFor(
+      async () => {
+        vitest.advanceTimersByTime(NETWORK_POLLING_INTERVAL);
+        for (const context of pollbookContexts) {
+          expect(
+            await context.localApiClient.getDeviceStatuses()
+          ).toMatchObject({
+            network: {
+              isOnline: true,
+              pollbooks: [
+                expect.objectContaining({
+                  status: PollbookConnectionStatus.WrongElection,
```

Comment:
is the pollbook package still close enough to the concept of an election that `WrongElection` still makes sense in this context?

Reply:
yes

Reply:
there is a ticket to improve the network status table so maybe we would want to differentiate between those cases when we get there but I doubt it I think its easier to just have it all in a wrong election bucket and start showing the hashes in that table